src/components/portrait/portrait-gravatar.d.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { PortraitSizes, PortraitShapes } from "./portrait";
+
+export interface PortraitGravatarProps {
+  /** The theme to use. */
+  theme?: object;
```

Comment:
I'm not sure if we have a better type for that?

Reply:
What are the valid options for this? Searching the rest of the PR, I can't see this prop being used.

Reply:
Valid options are themes from `src/style/themes`.

Reply:
I wonder if the right thing to do is to create a ts export for that folder and then we can reference those types here?

Reply:
Can we now import the theme interface to be used?